Update azure-subscription-limits-azure-resource-manager

nibhat opened the pull request:
Need to update the default no of VMs per subscription. It was listed as 20 per subscription which is wrong. It should be 10,000 per code. Confirmed with the CRP dev team

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

nibhat committed:
Update azure-subscription-limits-azure-resource-manager

Need to update the default no of VMs per subscription. It was listed as 20 per subscription which is wrong. It should be 10,000 per code. Confirmed with the CRP dev team
